src/coreclr/inc/corinfoinstructionset.h

```diff
@@ -128,41 +128,72 @@ enum CORINFO_InstructionSet
 struct CORINFO_InstructionSetFlags
 {
 private:
-    uint64_t _flags = 0;
+    uint64_t _flags[1] = {};
 public:
+    static const int FlagsArrSize = 1;
```

Comment:
Yes, will move `FlagsArrSize `. It is currently used for definition of `_flags` but I'm using the actual calculated value instead of the variable since we know at the time of generating this file what the value is. But I don't see a problem with switching to var as you suggested.

Reply:
Slight correction here - the reason `FlagsArrSize ` is currently public is because it's used from compiler.h. I see that you have another comment there. Will address that first before changing this.

Reply:
Changed